include/gsl/gsl

```diff
@@ -21,7 +21,6 @@
 #include <gsl/gsl_assert>    // Ensures/Expects
 #include <gsl/gsl_byte>      // byte
 #include <gsl/gsl_util>      // finally()/narrow()/narrow_cast()...
```

Comment:
```suggestion
#include <gsl/gsl_util>      // finally()/narrow()/narrow_cast()...
#include <gsl/multi_span>    // multi_span, strided_span...
```

We should add this back, the deprecation of the types should be sufficient. We can remove this header from <gsl/gsl> with the next major release (v4.0.0)